src/test/java/it/gov/pagopa/payment/dto/mapper/TransactionInProgress2TransactionOutcomeDTOMapperTest.java

```diff
@@ -137,11 +137,11 @@ private void commonAssertions(TransactionInProgress expected, TransactionOutcome
     Assertions.assertSame(expected.getMerchantId(), result.getMerchantId());
     Assertions.assertSame(expected.getMerchantFiscalCode(), result.getMerchantFiscalCode());
     Assertions.assertSame(expected.getVat(), result.getVat());
-    Assertions.assertSame(expected.getInitiativeId(), result.getInitiativeId());
+    Assertions.assertTrue(result.getInitiatives().contains(expected.getInitiativeId()));
```

Comment:
it's better to assert the equals, this way we know that it will contain just the expected initiative, in order the "equals" message print if failed is more explicative than the assertTrue
```suggestion
    Assertions.assertEquals(expected.getInitiativeId(), result.getInitiatives());
```